java/src/main/java/com/codurance/training/tasks/AddDelete.java

```diff
@@ -0,0 +1,22 @@
+package com.codurance.training.tasks;
+
+import java.io.PrintWriter;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+public class AddDelete  implements DeleteInterface{
```

Comment:
We are not adding anything here, we are just deleting
Rename AddDelete to better name